interfaces/prompting/requestprompts/requestprompts.go

```diff
@@ -38,6 +38,18 @@ import (
 	"github.com/snapcore/snapd/strutil"
 )
 
+var (
+	// initialTimeout is the duration before which prompts for a given user
+	// will expire if there has been no retrieval of prompt details for that
+	// user since the previous timeout, or if the user prompt DB was just
+	// created.
+	initialTimeout = 10 * time.Second
+	// initialTimeout is the duration before which prompts for a given user
```

Comment:
This seems copy pasted.

Reply:
Typo on my part, thanks for catching it :)